dstep/translator/Output.d

```diff
@@ -1062,14 +1060,13 @@ class ClassData : StructData
     string superclass;
     string[] interfaces;
 
-    HashSet!(string) propertyList;
+    bool[string] propertyList;
```

Comment:
It even has the same API, make the other changes related to `propertyList` unnecessary 😃,